CAT-89 default values incosistent number format

Tot333 opened the pull request:
- all bricks which had decimal values in preview now
  show int values

catrobatjenkins commented:
Can a catrobat member please verify this patch? If it doesn't look evil comment with "test this please" to start the testrun.

catrobatjenkins commented:
Can a catrobat member please verify this patch? If it doesn't look evil comment with "test this please" to start the testrun.

catrobatjenkins commented:
Can a catrobat member please verify this patch? If it doesn't look evil comment with "test this please" to start the testrun.

catrobatjenkins commented:
Can a catrobat member please verify this patch? If it doesn't look evil comment with "test this please" to start the testrun.

Tot333 commented:
thanks I did not see those two

I also changed those two now, please review again.

Tot333 commented:
- moved the function to FormulaBrick
- i'm still not satisfied with the new name of the function but it should be clear what it is for
- refactored the function itself so the string gets NOT created with ""+value

please review again

84n4n4 commented on catroid/src/main/java/org/catrobat/catroid/content/bricks/FormulaBrick.java:
```diff
@@ -170,4 +170,12 @@ public void storeDataForBackPack(Sprite sprite) {
 	}
+
+	public static String getStringFromFloatForBrickView(float value) {
+		if (((value % 1.0) == 0) ? true : false) {
```
seriously?
can you please check again what this inline-if ("?" ":") do here...

besides that id prefer 
value == Math.ceil(value)
to make it clear on what this actually does

84n4n4 commented on catroid/src/main/java/org/catrobat/catroid/physics/content/bricks/TurnRightSpeedBrick.java:
```diff
@@ -91,7 +91,8 @@ public View getView(Context context, int brickId, BaseAdapter baseAdapter) {
 		prototypeView = View.inflate(context, R.layout.brick_physics_turn_right_speed, null);
 		TextView textTurnRightSpeed = (TextView) prototypeView.findViewById(R.id.brick_turn_right_speed_edit_text);
-		textTurnRightSpeed.setText(String.valueOf(BrickValues.PHYSIC_TURN_DEGREES));
+		textTurnRightSpeed.setText(FormulaBrick.getStringFromFloatForBrickView(BrickValues
```
please make the method in formulaBrick protected and remove the static. then FormulaBrick.getStringFromFloatForBrickView can become getStringFromFloatForBrickView...

for the one usage in InternToExternGenerator please refactor this method.

cheidenreich commented:
please refactor according to @84n4n4 's comments and fix conflicts, closing this for now, feel free to reopen once it's properly implemented
